apierrors/errors.go

```diff
@@ -2,12 +2,19 @@ package apierrors
 
 import (
 	"errors"
+	"strconv"
 )
 
 //A lit of error messages for Permissions API
 var (
 	//ErrRoleNotFound is an error when the role can not be found in mongoDB
-	ErrRoleNotFound          = errors.New("role not found")
-	ErrInvalidQueryParameter = errors.New("invalid query parameter")
-	ErrLimitAndOffset        = errors.New("offset and limit must be positive or zero")
+	ErrRoleNotFound           = errors.New("role not found")
+	ErrInvalidPositiveInteger = errors.New("value is not a positive integer")
+	ErrLimitAndOffset         = errors.New("offset and limit must be positive or zero")
 )
+
+// ErrorMaximumLimitReached creates a unique error
+func ErrorMaximumLimitReached(m int) error {
+	err := errors.New("the maximum limit has been reached, the limit cannot be more than " + strconv.Itoa(m))
```

Comment:
I'm not requesting you make this change but just as an FYI you can do:

````
return fmt.Errorf("the maximum limit has been reached, the limit cannot be more than %d", m)
````
Functionally the same but one line of code. As I said I won't not approve based on this - just sharing knowledge 😄

Reply:
Thank you